### PR TITLE
2021 regionals membership updates

### PR DESCRIPTION
**Pull request by adamisch**
Waiting on Theresa Nyowheoma to provide some biographical information.

### CODE REVIEW

File: members.md [hunk 1]
```diff
@@ -845,29 +883,25 @@ Provisional members are community members who have ably assisted ACF in the past
 </td>
 </tr>
 <tr>
-<td>Olivia Murton</td>
+<td>Arjun Nageswaran</td>
 <td>
 <ul>
-<li>PhD, Harvard University</li>
-<li>Alumna, MIT</li>
+<li>Stevenson High School</li>
```
Comment (hftf) [inline members.md:889]: is this the preferred affiliation?

File: members.md [hunk 2]
```diff
@@ -765,16 +804,15 @@ Provisional members are community members who have ably assisted ACF in the past
...
 <ul>
-<li>Alumnus, Yale University</li>
+<li>Alumna, Yale University</li>
 </ul>
 </td>
 <td>
 <ul>
-<li>2021 Site Coordinator (interim)</li>
-<li>2020 ACF Fall and Regionals Site Tournament Director</li>
+<li>2019 ACF Regionals Editor</li>
```
Comment (hftf) [inline members.md:815]: 2021?

File: members.md [hunk 3]
```diff
@@ -524,6 +539,20 @@ Full members are ACF's editors, officers, and tournament directors. They vote in
 <li>2019 ACF Fall Editor</li>
 </ul>
 </td>
+</tr>
+ <tr>
```
Comment (hftf) [inline members.md:543]: (leading space can be removed)

File: members.md (hunk 2 above)
Comment (adamisch) [reply members.md]: fixed

**Pull request merged**